libgstd/libgstd.h

```diff
@@ -121,6 +101,48 @@ gstd_stop (GstD * gstd);
 void
 gstd_free (GstD * gstd);
 
+/**
+ * gstd_create:
+ * 
+ * @gstd: A valid gstd instance allocated with gstd_new()
+ * @uri: Path to the resource in which the action will be
+ * applied in low level CRUD syntax 
+ * @name: Name of the resource to create
+ * @description: Description of the resource to create
+ * 
+ * A new Create call of the argument with the description
+ *
+ * Returns: GstdReturnCode indicating success or fail
+ */
+GstdReturnCode gstd_create (GstD *gstd, const gchar *uri, const gchar *name, const gchar *description);
```

Comment:
Let's try to stick to the pattern in the other declartions:
```
GstdReturnCode
gstd_create (GstD *gstd, const gchar *uri, const gchar *name, const gchar *description);
```
ditto for the other declartions you have added